### PR TITLE
fix(uploader): increase chunked upload threshold to 100MB

### PR DESCRIPTION
**Pull request by stevenzyang**
Matches the pre-react behavior of uploader.

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @stevenzyang, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: src/elements/content-uploader/ContentUploader.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ type State = {
     view: View,
 };
 
-const CHUNKED_UPLOAD_MIN_SIZE_BYTES = 52428800; // 50MB
+const CHUNKED_UPLOAD_MIN_SIZE_BYTES = 104857600; // 100MB
```
Comment (Visylvius) [inline src/elements/content-uploader/ContentUploader.js]: Could this be moved to a constants file?

Comment (stevenzyang) [reply src/elements/content-uploader/ContentUploader.js]: Outside the scope of this PR

**Pull request merged**